crates/tuono/src/app.rs

```diff
@@ -176,15 +177,18 @@ impl App {
 
     pub fn build_react_prod(&self) {
         if !Path::new(BUILD_JS_SCRIPT).exists() {
-            eprintln!("Failed to find the build script. Please run `npm install`");
+            error!("Failed to find the build script. Please run `npm install`");
```

Comment:
<img width="675" alt="image" src="https://github.com/user-attachments/assets/95d36d04-128d-476a-894a-fa418afa326f" />

IMHO the error should appear on a different line from the spinner.
Also the spinner should be removed or replaced with something like "⛔️ Build failed"

Ref: https://github.com/tuono-labs/tuono/pull/560#discussion_r1955652116

Reply:
Seems your CLI is using a set of font similar to noto:
- https://emojipedia.org/cross-mark#designs
- https://emojipedia.org/noto-emoji

There isn't much to do. When working on https://github.com/orgs/tuono-labs/projects/2?pane=issue&itemId=98195527 we should consider to change message color or use something else to better identify the type of the message.